common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -74,23 +74,38 @@ class @Problem
-        progress_template = ngettext('(%(earned)s/%(possible)s point)', '(%(earned)s/%(possible)s points)', possible)
+
+        if graded == "True" and possible != 0
+            # This comment needs to be on one line to be properly scraped for the translators. Sry for length.
+            `// Translators: %(earned)s is the number of points earned. %(total)s is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points)`
+            progress_template = ngettext('%(earned)s/%(possible)s point earned (graded)', '%(earned)s/%(possible)s points earned (graded)', possible)
+        else
+            # This comment needs to be on one line to be properly scraped for the translators. Sry for length.
+            `// Translators: %(earned)s is the number of points earned. %(total)s is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points)`
+            progress_template = ngettext('%(earned)s/%(possible)s point earned (ungraded)', '%(earned)s/%(possible)s points earned (ungraded)', possible)
         progress = interpolate(progress_template, {'earned': earned, 'possible': possible}, true)
 
     # Render 'x point(s) possible' if student has not yet attempted question
-    if status == 'none' and detail? and (jQuery.type(detail) == "string") and detail.indexOf('/') > 0
-        a = detail.split('/')
-        possible = parseFloat(a[1])
-        `// Translators: %(num_points)s is the number of points possible (examples: 1, 3, 10). There will always be at least 1 point possible.`
-        progress_template = ngettext("(%(num_points)s point possible)", "(%(num_points)s points possible)", possible)
+    if status == 'none' or status == 0
+        if detail? and (jQuery.type(detail) == "string") and detail.indexOf('/') > 0
+            a = detail.split('/')
+            possible = parseFloat(a[1])
+        else
+            possible = 0
+
+        if graded == "True" and possible != 0
+            `// Translators: %(num_points)s is the number of points possible (examples: 1, 3, 10).`
+            progress_template = ngettext("%(num_points)s point possible (graded)", "%(num_points)s points possible (graded)", possible)
+        else
+            `// Translators: %(num_points)s is the number of points possible (examples: 1, 3, 10).`
+            progress_template = ngettext("%(num_points)s point possible (ungraded)", "%(num_points)s points possible (ungraded)", possible)
         progress = interpolate(progress_template, {'num_points': possible}, true)
 
     @$('.problem-progress').html(progress)
```

Comment:
We should make sure this text is escaped. I think you can just switch to using ".text" instead of ".html".

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html